src/components/ContentPreview/ContentPreview.js

```diff
@@ -828,14 +847,11 @@ class ContentPreview extends PureComponent<Props, State> {
         const fileId =
             typeof fileOrId === 'object' ? fileOrId.id || '' : fileOrId;
 
+        this.setState({
+            currentFileId: fileId,
```

Comment:
Should navigate below be as a callback? Or does it not matter

Reply:
It doesn't matter